RPIPs/RPIP-HoustonHotfix.md

```diff
@@ -0,0 +1,93 @@
+---
+rpip: TBC
+title: Houston Hotfix
+description: This proposal defines what is included in the Houston hotfix.
+author: langers (@langers)
+discussions-to: 
+status: Draft
+type: Protocol
+category: Core
+created: 2024-09-09
+---
+
+## Abstract
+
+The Houston hotfix does not introduce new features but does include important changes to existing features that:
+
+- Fix a discovered bug with finalising legacy minipools
+- Resolves Immunefi bug bounty submissions regarding the onchain voting system
+- Improves Protocol DAO quorum default guardrail value (currently set too high)
+- Fixes a couple of issues with Protocol DAO parameters
+- Refactored how the onchain voting system burns a veto'd proposer bond - now actually burns the RPL rather than transferring it to 0x0 address
```

Comment:
Change "Refactored" to "Modified" as refactored implies there is no change to external behaviour